example-config.yml

```diff
@@ -320,7 +320,7 @@ itinerary:
   #  electronicRegular: "SmartCard Fare"
   #  student: "Student Fare"
   # One fare will always be shown by default
-  #defaultFareKey: electronicRegular
+  defaultFareType: { mediumId: null, riderCategoryId: null }
```

Comment:
This is an awesome default, but we need it to also apply if the key is not set!

Reply:
I'm seeing the fare in the metro itinerary, but not in the batch ui/calltaker ui or the trip details!

Reply:
Fares were removed from trip-details as of version 5.0, right? Btw @daniel-heppner-ibigroup you could use the non-alpha trip-details 5.0.0.

Reply:
Oh I wasn't aware...

Reply:
Oh yes I know but I'm still not seeing "new-school" fares in the trip-details when I am seeing them in the metro itinerary. I think something is wrong with the data being passed around

Reply:
Fixed the bug with a new alpha version of trip-details. 🙃